app/src/hooks/reversiGame.ts

```diff
@@ -20,6 +27,7 @@ export type ReversiGameType = {
   getScore: (disc: DiscType) => number;
   reset: () => void;
   revertMove: (count?: number) => boolean;
+  moveScores: Array<MoveScore>;
```

Comment:
ask
```suggestion
  moveScores:MoveScore[];
```
でいいと思いましたがいかがですか
他とある程度揃っていたほうが良いと思いました